packages/expo-router/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unpublished
 
+- Fix false positive redirect deprecation since version 2.0.1 when using `<Screen />`. ([#23932](https://github.com/expo/expo/pull/23932) by by [@sync](https://github.com/sync))
```

Comment:
```suggestion
- Fix false positive redirect deprecation since version 2.0.1 when using `<Screen />`. ([#23932](https://github.com/expo/expo/pull/23932) by [@sync](https://github.com/sync))
```